ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/ConfigManagementServiceIT.scala

```diff
@@ -15,6 +15,7 @@ final class ConfigManagementServiceIT extends LedgerTestSuite {
     "CMSetAndGetTimeModel",
     "It should be able to get, set and restore the time model",
     allocate(NoParties),
+    isolated = false,
```

Comment:
I actually interpreted `isolated` as in "this test is run at the end in isolation". Maybe rename it to something like `runExclusively`?

Reply:
I was hoping for some feedback on naming. :bow:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6651/commits/36b11043b20a91f3d8878151ff9826c270848357